### PR TITLE
Updated readme with step by step guide for CRC

### PR DESCRIPTION
**Pull request by sbryzak**
Fixes https://issues.redhat.com/browse/CRT-952

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -44,15 +46,93 @@ If you still don't know what to do with e2e tests in some use-cases, go to <<Wha
...
+9. Log into the cluster by executing the `oc login` command as shown in the resulting output from running `crc start` in the previous step.  The output should look something like this:
++
+image::crc_oc_login.png[align=center]
+
+10. Check out the CodeReady Toolchain `host-operator` and `member-operator` repositories and ensure that they are located at the same directory level as the `toolchain-e2e` repository.
++
+* https://github.com/codeready-toolchain/host-operator
+* https://github.com/codeready-toolchain/member-operator
++
+11. Change into the `toolchain-e2e` directory, and execute the tests.  It is recommended that you direct console output to a text file, as the test output is of significant length and is likely to be longer than your console buffer:
```
Comment (MatousJobanek) [inline README.adoc:125]: Personally, I've configured my terminal to have an unlimited buffer for the scrolling :-)

**Review by MatousJobanek — APPROVED** [review]
This guide is great :+1: :star_struck: 
Thanks for it @sbryzak

File: README.adoc (hunk 1 above)
Comment (alexeykazakov) [reply README.adoc:125]: I output to a file too. It's easier to search and navigate in a file. But it's all personal preferences :)

**Review by alexeykazakov — COMMENTED** [review]
Great step-by-step guide!
But I would move it to a dedicated CRC.adoc file and just add a link to the README.adoc to avoid too long README.

File: README.adoc (hunk 1 above)
Comment (MatousJobanek) [reply README.adoc:125]: Yes, sure - I didn't mean that it should be removed :-). I just mentioned it in case Shane wouldn't be aware of this option :wink: 
The README part can stay as it is :+1:

File: README.adoc [hunk 2]
```diff
@@ -44,15 +46,93 @@ If you still don't know what to do with e2e tests in some use-cases, go to <<Wha
...
++
+`make test-e2e-local >testoutput.txt`:
```
[the remark this answers is not in the document]
Comment (sbryzak) [reply README.adoc:127]: This was intentional, however I can update the readme to cover both cases.

File: testsupport/init.go [hunk 3]
```diff
@@ -95,7 +95,7 @@ func WaitForDeployments(t *testing.T, obj runtime.Object) (*framework.Context, *
 func getMemberAwaitility(t *testing.T, f *framework.Framework, hostAwait *wait.HostAwaitility, namespace string) *wait.MemberAwaitility {
 	memberClusterE2e, err := hostAwait.WaitForToolchainClusterWithCondition("e2e", namespace, wait.ReadyToolchainCluster)
 	require.NoError(t, err)
-	memberConfig, err := cluster.NewClusterConfig(f.Client.Client, &memberClusterE2e, 3*time.Second)
+	memberConfig, err := cluster.NewClusterConfig(f.Client.Client, &memberClusterE2e, 5*time.Second)
```
Comment (alexeykazakov) [inline testsupport/init.go:98]: This should not be part of this PR, should it?

File: tiers/checks.go [hunk 4]
```diff
@@ -701,7 +701,20 @@ func numberOfNetworkPolicies(number int) namespaceObjectsCheck {
 		assert.Len(t, nps.Items, number)
 	}
 }
-
+/*
```
Comment (alexeykazakov) [inline tiers/checks.go:704]: And this either.

**Review by alexeykazakov — APPROVED** [review]
Looks good. Thanks for addressing the comments. But let's clean up the changes from another PR.

File: tiers/checks.go (hunk 4 above)
Comment (sbryzak) [reply tiers/checks.go]: Fixed

File: testsupport/init.go (hunk 3 above)
Comment (sbryzak) [reply testsupport/init.go]: It does need to be increased as it's intermittently causing e2e tests in CRC to fail, however I can move to a different PR.

File: README.adoc [hunk 5]
```diff
@@ -44,15 +46,93 @@ If you still don't know what to do with e2e tests in some use-cases, go to <<Wha
 * yq (python-yq from https://github.com/kislyuk/yq#installation, other distributions may not work)
 * docker (if building from source)
 
-==== Minishift or CRC (CodeReady Containers)
+==== Minishift
```
[the remark this answers is not in the document]
Comment (sbryzak) [reply README.adoc]: Done
